images/tier2/saved.svg

```diff
@@ -0,0 +1,4 @@
+<svg xmlns="http://www.w3.org/2000/svg" height="24" width="24" viewBox="0 0 24 24">
```

Comment:
This is a bit confusing with the `save` icon. Can we rename them both to be `save-filled` and `save-hollow`? Similar to how we're doing pinned.